resi.py

```diff
@@ -1,40 +1,89 @@
+    tipe = "4DX"
+    harga = 110000
+else:
+    tipe = "Premium"
+    harga = 350000
+    
+print(f"Jumlah Tiket:", data['jumlah tiket'])
+print(f"Daftar Tempat Duduk:", data['daftar tempat duduk'])
+
+
+#   Resi -- Booking Tiket Bioskop --   #
 
 def main():
     
-    print("================================================")
-    print("                Cinegaje Bioskop                ")
-    print("------------------------------------------------")
-    print(" Detail Transaction     DD-MM-YYYY WIB/WIT/WITA ")
-    print("                                                ")
-    print(" ------------------                             ")
-    print(" |                |  Fast X                     ")
-    print(" |                |                             ")
-    print(" |                |  Date: Thursday,DD-MM-YYYY  ")
-    print(" |                |  Time: 10:00 WIB/WIT/WITA   ")
-    print(" |                |                             ")
-    print(" |                |                             ")
-    print(" |                |  Booking Code: 9432940      ")
-    print(" |                |                             ")
-    print(" |                |                             ")
-    print(" |                |                             ")
-    print(" ------------------                             ")
-    print("                                                ")
-    print(" Transaction Ref :   Cinegaje/RES/DD-YYYY/8349  ")
-    print(" Seats           :   A11                        ")
-    print(" Tipe Bioskop    :   Reguler/Deluxe 4DX Premium ")
-    print(" Total Tickets   :   5                          ")
-    print(" Ticket Price    :   Rp100.000,00               ")
-    print(" Promo/Diskon    :   Rp0                        ")
-    print(" Payment         :   GOPAY/OVO/BCA/TUNAI/KREDIT ")
-    print("                                                ")
-    print(" Total Payment   :   Rp100.000,00               ")
-    print("                                                ")
-    print("                                                ")
-    print("              ==Selamat Menonton==              ")
-    print("                    ==Enjoy==                   ")
-    print("                                                ")
-    print("================================================")
+    print(f"================================================")
+    print(f"                Cinegaje Bioskop                ")
+    print(f"------------------------------------------------")
+    print(f" Detail Transaction     DD-MM-YYYY WIB/WIT/WITA ")
+    print(f"                                                ")
+    print(f" ------------------                             ")
+    print(f" |                |  Fast X                     ")
+    print(f" |                |                             ")
+    print(f" |                |  Date: Thursday,DD-MM-YYYY  ")
+    print(f" |                |  Time: 10:00 WIB/WIT/WITA   ")
```

Comment:
Tolong untuk date dan time nya bisa sync dgn datetime saat ini ya, bisa dimasukkin database atau pakai `import datetime`

Reply:
CONTOH:
Taruh import datetime dulu diawal:
`from datetime import date`
`from datetime import datetime`

Kemudian, buat seperti ini:
```suggestion
    now = datetime.now()
    current_time = now.strftime("%H:%M:%S")
    
    print(f" |                |  Date: {date.today()}")
    print(f" |                |  Time: {current_time}")
```